container-registry-ami-builder/images/README.md

```diff
@@ -1,8 +1,10 @@
-# Images folder
 ## Description
-This is a folder to store all your docker images. Make sure to store them as tar files to initiate a harbor ami build
+You can pull all images in your local environment and save them as tar files. Then copy all tar files to the `~/aws-snow-tools-for-eks-anywhere/container-registry-ami-builder/images` directory on the AL2 instance. The AMI build process will iterate over the `images` folder and upload the tar files to the target AMI.
 
-1. `docker pull` all the images and run `docker save IMAGE_NAME > IMAGE_NAME.tar` to save it as tar file
-2. Copy all tar files to your al2 instance under images folder before running build.sh script during the ami build process.
-> `scp -i your-pemKey <path-to-your-tar-file/your-tar-file> ec2-user@<instance ip>:~/AwsSnowHarborAmiBuild/images/`
-3. build.sh script will automatically read images folder and get your tar file to build up the harbor ami 
+1. `docker pull` all the images and run `docker save IMAGE_NAME > IMAGE_NAME.tar` to save them as tar files
+2. Copy all tar files to your AL2 instance under the `~/aws-snow-tools-for-eks-anywhere/container-registry-ami-builder/images` folder before running `build.sh` script during the AMI build process.
+
+###### Note: `<path-to-key>` is the path to your ssh key pair file of your Harbor AMI build instance
+```
+scp -i <path-to-key>/<key-file-name> <path-to-your-tar-file/your-tar-file> ec2-user@<public-IPv4-address>:~/aws-snow-tools-for-eks-anywhere/container-registry-ami-builder/images/
```

Comment:
```suggestion
scp -i <path-to-key> <path-to-your-tar-file/your-tar-file> ec2-user@<public-IPv4-address>:~/aws-snow-tools-for-eks-anywhere/container-registry-ami-builder/images/
```
`<path>/*.tar` might be better, which will copy all the tar files with a single command, could you try it out to see if it will work or not?

Reply:
I don't think it's feasible since user may have other tar files in their folder which they don't want to upload.

Reply:
then do you need to mention "need to run this command for every image tar file" ?